Misc/NEWS.d/next/Core and Builtins/2021-02-15-13-41-14.bpo-40170.r2FAtl.rst

```diff
@@ -0,0 +1,2 @@
+Convert :c:func:`PyDescr_IsData` macro to function. Patch by Erlend E.
```

Comment:
Can you explain the reason? I understand that it's to avoid accessing directly to PyTypeObject members (hide implementation details).

Reply:
Sure, I've expanded the NEWS entry similar to other PR's in this bpo :)